.github/workflows/lint.yml

```
@@ -47,3 +47,11 @@ jobs:
         python tools/lint.py --diff-against origin/$GITHUB_BASE_REF
         python tools/unicode-check.py
         python tools/check_test_name.py
+
+    - name: Check that Python.h is first in any file including it.
+      shell: bash
+      run: |
+        python tools/check_python_h_first.py $(find scipy \
+            \( -name highspy -o -name boost \) -prune \
+            -o -type f \( -iname \*.[ch] -o -name \*.[ch]pp -o -name \*.[ch]xx \
+                -o -name \*.cc -o -name \*.hh \) -print)
```

```suggestion
        python tools/check_python_h_first.py 
```
was how I tested the most recent changes locally; not *quite* the same as `python dev.py lint`, but the differences are primarily in whether the first include file must be `Python.h` or a file including `Python.h`